chapters/statements.tex

```diff
@@ -416,7 +416,7 @@ \subsubsection{Restrictions on Where a When-statement may occur}\label{restricti
 \end{lstlisting}
 \end{example}
 
-\subsubsection{Restrictions on Statements within When-Statements}\label{restrictions-on-statements-within-when-statements}
+\subsubsection{Statements within When-Statements}\label{restrictions-on-statements-within-when-statements}\label{statements-within-when-statements}
```

Comment:
Another inconsistent capitalization: "When-Statement" while on line 395 it is "When-statement"

Reply:
Hmm... Good point, I had to investigate and it seems that when you capitalize it as a title it should be 'When-Statement'. https://titlecaseconverter.com/blog/capitalizing-hyphenated-words/

Reply:
Should now be done.

Reply:
Looks good. Resolving.